view/frontend/web/css/product-style.css

```diff
@@ -0,0 +1,32 @@
+@media (max-width: 767px) {
+    .catalog-product-view .hydrated center {
+        text-align:right;
+        background: #fff;
+        position: fixed;
+        bottom: 0;
+        left: 0;
+        z-index: 9999;
+        width: 50%;
+    }
+    .catalog-product-view .hydrated center ion-button {
+        margin-bottom: 10px;
+        margin-left: 5px;
+        width: 96% !important;
+        border-radius: 0px!important;
+        height: 50px!important;
+    }
+    .box-tocart .actions {
```

Comment:
This might be tricky since the selectors are pretty generic and might have unexpected implications when clients implement the plugin. I would suggest only applying styling to the button itself and try to avoid big changes to the surrounding elements

Reply:
It's just move the Add to cart and Amwal button to end of the page in mobile display. in case someone need to make any changes they can overwrite a file by the theme files.

Reply:
![Screenshot 2023-09-26 at 10 07 13](https://github.com/amwal-tech/amwal-magento/assets/141551822/5bac6949-0ba6-4ed0-b1b6-c46eb8fb9c60)